### PR TITLE
Support for abbreviate

### PR DESCRIPTION
**Pull request by hybrist**
cc @dbushong

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -26,9 +28,10 @@ var GROUP = new RegExp([
   '<([^>]*)>', // We capture the actual range expression
   ')?',
   // The trailing part is optional
-  '([^,;\\s]+)?',
+  '([^,;\\s]*)',
   ')', // end of actual group
 ].join(''), 'g');
+var HOST = /^([^\d,;\s<>]+)([\d]*)([^,;\s<>]*)$/;
```
Comment (dbushong) [inline index.js]: Superfluous `[]` around the `\d` in the middle?  Did you want to explicitly fail on double-numbered hosts? (`foo14sub3.tld`)

Comment (hybrist) [inline index.js]: Removed the `[]`. I don't think we can fail on double-numbered hosts. E.g. things with custom tlds like `foo-app3.tld1` are valid hostnames we'd need to support.

**Comment by hybrist** [comment]
Came up with another edge case while on the bus. `[ "a1", "a" ]` would result in `[ "a1" ]`. Fixed now.

**Comment by hybrist** [comment]
`expand(abbr([]))` is now `[]` and `abbr([])` is `""`.

**Comment by dbushong** [comment]
LGTM - since you're being fairly rigorous about validation, maybe also a line to make sure you were passed an array and not, e.g., a string (or `null`)

**Comment by hybrist** [comment]
Good point! Added a quick check w/ `TypeError` that the argument is an array.

**Comment by dbushong** [comment]
:sheep: it!

**Comment by hybrist** [comment]
v1.1.0

**Pull request merged**